build/teamcity-bazel.sh

```diff
@@ -0,0 +1,18 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+source "$(dirname "${0}")/teamcity-support.sh"
+
+tc_prepare
+
+export TMPDIR=$root/artifacts
```

Comment:
Where does `$root` get set? Does that come from TeamCity itself?

Reply:
Added comment.

Reply:
Ah! That's right. I had forgotten it was from teamcity-support.sh and didn't think to check there. Thanks